### DISCUSSION TITLE
[Feature]: UX improvements in albums

### DISCUSSION THREAD

**Discussion by rounakdatta** [immich-app/immich#1643] [Feature Request]
### Feature detail

Here are some of the things that might significantly improve the user experience while using albums:
- Ability to merge albums
- Move images in a particular album to another album
- Ability to hide particular images while sharing
- Ability to screenshot a scene from a video and set it as the album cover
- Sharing an album publicly
- Add a description to album
- Select all button while selecting images in an album

If the aforementioned features seem interesting, then we can create sub-issues.

### Platform

Web

**Comment by alextran1502** [comment]
Love all the ideas, thank you

**Comment by nep2ner** [comment]
I think it would also be cool to be able to include the contents of select shared albums in the main photo stream.

**Comment by stuomas** [comment]
I came to write feature idea about album descriptions, but it was already included in this basically. I would like the ability to write a proper story or diary style entry about albums, to make like a visual diary. Would be great if it allowed proper text formatting (at least dividing to paragraphs, if not full blown markdown/html)

Long live immich! Love the project.

**Comment by enboig** [comment]
Could there be a "feature request" poll/ranking with all features individual?
I would really like the:
- Merge albums (typo made me create 2 albums...)
- Move items from one album to another
- Being able to add images to multiple albums at once (tag style?)
- Being able to "add to albums + archive" in a single step; right when I add an image to an album, they are unselected and I have to select them again to archive.

**Comment by Behem07h** [comment]
mass selecting images would be great. would make this https://github.com/immich-app/immich/discussions/1678 more tolerable at least

**Comment by bo0tzz** [comment]
Most of these are now implemented. For those that are still missing, please create new requests if none exist yet.
